railties/lib/rails/generators/rails/app/templates/github/ci.yml.tt

```diff
@@ -58,6 +58,16 @@ jobs:
           ruby-version: .ruby-version
           bundler-cache: true
 
+      - name: Prepare RuboCop cache
+        uses: actions/cache@v4
+        env:
+          DEPENDENCIES_HASH: ${{ hashFiles('.ruby-version', '**/.rubocop.yml', 'Gemfile.lock') }}
+        with:
+          path: ~/.cache/rubocop_cache/
```

Comment:
Usually I do use custom path like `/tmp/rubocop` and call `bin/rubocop` with `--cache-root /tmp/rubocop` to keep it bulletproof for future changes of default cache location.

Reply:
That would make it more resistant to possible cache location changes, yes. I do this on some projects, but I use [`RUBOCOP_CACHE_ROOT`](https://docs.rubocop.org/rubocop/usage/caching.html#cache-path) env variable which I set up on job level so it's not duplicated, e.g. (truncated example):
```yml
jobs:
  rubocop:
    env:
      RUBOCOP_CACHE_ROOT: tmp/rubocop
    steps:
      - uses: actions/cache@v4
        path: ${{ env.RUBOCOP_CACHE_ROOT }}
      - name: RuboCop
        run: bundle exec rubocop
```

Reply:
I just remembered that users can also override cache directory in their `.rubocop.yml` (`AllCops.CacheRootDirectory` property), so we definitely have to set something like this to make sure cache is in the right location on CI. I'll update the PR.

Reply:
Updated.